components/canvas/webgl_paint_task.rs

```diff
@@ -2,36 +2,44 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-use canvas_msg::{CanvasWebGLMsg, CanvasCommonMsg, CanvasMsg};
+use canvas_msg::{CanvasMsg, CanvasWebGLMsg, CanvasCommonMsg};
```

Comment:
You never know and it makes the patch noisier and harder to review. The only free bug code is the code you never write.